src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigurationFilesControl.xaml

```diff
@@ -0,0 +1,67 @@
+<UserControl x:Class="NuGet.PackageManagement.UI.Options.ConfigurationFilesControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:options="clr-namespace:NuGet.PackageManagement.UI.Options"
+             Background="{DynamicResource {x:Static SystemColors.ControlBrushKey}}"
+             Foreground="{DynamicResource {x:Static SystemColors.ControlTextBrushKey}}"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800">
+  <UserControl.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <options:SharedResources />
+      </ResourceDictionary.MergedDictionaries>
+      <Style TargetType="ListViewItem">
+        <EventSetter Event="MouseDoubleClick" Handler="ListViewItem_MouseDoubleClick"/>
+      </Style>
+    </ResourceDictionary>
+  </UserControl.Resources>
+  <UserControl.CommandBindings>
+    <CommandBinding
+      Command="{x:Static nuget:PackageManagerControlCommands.OpenExternalLink}"
+      Executed="ExecuteOpenExternalLink" />
+  </UserControl.CommandBindings>
+  <Grid>
+    <Grid.RowDefinitions>
+      <RowDefinition Height="auto" />
+      <RowDefinition Height="*" />
+      <RowDefinition Height="auto" />
+    </Grid.RowDefinitions>
+    <TextBlock Grid.Row="0" Text="{x:Static nuget:Resources.VSOptions_Label_ConfigurationFiles}"/>
+    <ListView
+      Name="_configurationFiles"
+      Grid.Row="1"
+      SelectedItem="{Binding SelectedPath}"
+      ItemsSource="{Binding ConfigurationFilesCollection}"
+      AutomationProperties.Name="{x:Static nuget:Resources.VSOptions_Label_ConfigurationFiles}" />
+    <Grid Grid.Row="2">
+      <Grid.ColumnDefinitions>
+        <ColumnDefinition Width="*"></ColumnDefinition>
+        <ColumnDefinition Width="auto"></ColumnDefinition>
+      </Grid.ColumnDefinitions>
+      <Grid Grid.Column="0">
+        <Grid.RowDefinitions>
+          <RowDefinition Height="auto" />
+          <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+        <TextBlock Grid.Row="0" TextWrapping="WrapWithOverflow" Text="{x:Static nuget:Resources.Text_ApplyingConfigurationFiles}"/>
+        <TextBlock Grid.Row="1" TextWrapping="WrapWithOverflow">
+          <Hyperlink
+              NavigateUri="https://learn.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#how-settings-are-applied"
```

Comment:
```suggestion
              NavigateUri="https://learn.microsoft.com/nuget/consume-packages/configuring-nuget-behavior#how-settings-are-applied"
```

Docs links should never have the language code

 Honestly, considering that the docs site used to be docs.microsoft.com and is now learn.microsoft.com, I'm not sure it's a good idea for us to hardcode a link to the current hostname, in case it changes again. Plus, hardcoding it assumes that we'll never change the rest of the URL either. I think we should use an aka..ms link so that if the docs URL ever changes in the future, we can update the aka.ms link, and all versions of VS that use the link will Just Work.

Reply:
I agree with @zivkan. I'm used to using fwlinks in code to link to docs. I suppose an aka.ms would work as well. Either way, we should use a link that is redirected so we can update it if it every changes without having to ship a code update.